pkg/controller/endpointslice/endpointslice_controller.go

```diff
@@ -274,10 +284,11 @@ func (c *Controller) Run(ctx context.Context, workers int) {
 		return
 	}
 
-	logger.V(2).Info("Starting worker threads", "total", workers)
+	logger.V(2).Info("Starting worker threads", "total", workers + 1)
 	for i := 0; i < workers; i++ {
 		go wait.Until(func() { c.worker(logger) }, c.workerLoopPeriod, ctx.Done())
 	}
+	go wait.Until(func() {c.topologyQueueWorker(logger)}, c.workerLoopPeriod, ctx.Done())
```

Comment:
Nitpick: the linter is probably going to point out this line.

Reply:
Done